Makefile

```diff
@@ -86,7 +86,7 @@ package-custom-resources-clean:
 
 .PHONY: run-unit-test
 run-unit-test:
-	go test -race -count=1 -coverprofile=${COVERAGE} ${PACKAGES}
+	go test -coverprofile=${COVERAGE} ${PACKAGES}
```

Comment:
why remove race detector?

Reply:
It was slowing down unit tests! I thought that I can send a separate CR where in the `ci` workflow we run the test with and without race detector

Reply:
fair enough :) we can leave it off for now, but the `ci` approach sounds good to me.